### PR TITLE
implement inverter.pvwatts_multi

### PR DESCRIPTION
**Pull request by cwhanse**
~~- [x] Closes #xxxx~~
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Tests added
 - [x] Updates entries to [`docs/sphinx/source/api.rst`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/api.rst) for API changes.
 - [x] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/master/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

Adds `inverter.pvwatts_multi` to extend `inverter.pvwatts` to the case of multiple MPP inputs. Discussion [here](https://github.com/pvlib/pvlib-python/pull/1076#discussion_r539475170).

### CODE REVIEW

File: docs/sphinx/source/whatsnew/v0.8.1.rst [hunk 1]
```diff
@@ -17,8 +17,8 @@ Enhancements
   option to :py:class:`~pvlib.modelchain.ModelChain`. (:pull:`1042`) (:issue:`1073`)
 * Added :py:func:`pvlib.temperature.ross` for cell temperature modeling using
   only NOCT. (:pull:`1045`)
-* Added :py:func:`pvlib.inverter.sandia_multi` for modeling inverters with
-  multiple MPPTs (:issue:`457`, :pull:`1085`)
+* Added :py:func:`pvlib.inverter.sandia_multi` and :py:func:`pvlib.inverter.pvwatts_multi`
+  for modeling inverters with multiple MPPTs (:issue:`457`, :pull:`1085`, :pull:`1103`)
```
Comment (kanderso-nrel) [inline docs/sphinx/source/whatsnew/v0.8.1.rst:21]: 1103 -> 1106

File: pvlib/inverter.py [hunk 2]
```diff
@@ -348,8 +348,9 @@ def pvwatts(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
 
     Parameters
     ----------
-    pdc: numeric
-        DC power. Same unit as ``pdc0``.
+    p_dc : tuple, list or array of numeric
+        DC power on each MPPT input of the inverter. If type is array, must
+        be 2d with axis 0 being the MPPT inputs. Same unit as ``pdc0``.
```
Comment (kanderso-nrel) [inline pvlib/inverter.py:353]: This makes sense for `pvwatts_multi`, but is it also true here in `pvwatts`?

File: pvlib/inverter.py [hunk 3]
```diff
@@ -396,6 +397,39 @@ def pvwatts(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
...
+        and is included here for flexibility. [unitless]
+
+    Returns
+    -------
+    power_ac: numeric
+        AC power.  Same unit as ``pdc0``.
+
+    See also
+    --------
+    pvlib.inverter.pvwatts
```
Comment (kanderso-nrel) [inline pvlib/inverter.py:428]: Consider also adding a See Also link to `pvwatts_multi` from `pvwatts`.

**Review by kanderso-nrel — APPROVED** [review]
LGTM

File: pvlib/inverter.py (hunk 2 above)
Comment (cwhanse) [reply pvlib/inverter.py:353]: The perils of cut and paste

Comment (kanderso-nrel) [reply pvlib/inverter.py]: Oh, `p_dc` vs `pdc` discrepancy between docstring and signature for both functions

**Review by wholmgren — APPROVED** [review]

**Pull request merged**